### PR TITLE
ports goofball foamchange except makes it more unfair

### PR DESCRIPTION
**Pull request by ghost**
ports the goofball foamchange except makes it even more OP
#### Changelog

:cl:
rscadd: FOAM IS NOT ASS. Credit to goofball.
/:cl:

### CODE REVIEW

**Comment by AsV9** [comment]
So what does this do then?
Please explain more.

File: code/game/objects/effects/effect_system/effects_foam.dm [hunk 1]
```diff
@@ -147,7 +155,7 @@
 		location = get_turf(loca)
 
 	amount = round(sqrt(amt / 2), 1)
-	carry.copy_to(chemholder, 4*carry.total_volume) //The foam holds 4 times the total reagents volume for balance purposes.
+	carry.copy_to(chemholder, 6*carry.total_volume) //The foam holds 4 times the total reagents volume for balance purposes.
```
Comment (Super3222) [inline code/game/objects/effects/effect_system/effects_foam.dm:158]: foam now creates 6 times the total reagents..

**Comment by Super3222** [comment]
This seems like an odd change. What's the reason behind it?

**Comment by ghost** [comment]
Goofball changed it so that foam made 5 times total reagents, i made it 6 instead

Reason-clothes absorb the majority of foam, even basic clothes, so gas is 4000x superior. This brings foam on par with gas because foam will actually apply it's reagents.

The issue with foam is that it divides it's reagents over the tiles it covers and then goes throgh clothes so a 50u acid foam will end uo just being 3u acid. You need to hse tiny amounts of foam to get an effect so smoke was superior

Now 50u acid will apply properly to people in it

**Comment by BOBsHERE** [comment]
Awesome. Basically the reason I never bothered with foam grenades.

**Comment by Super3222** [comment]
Went ahead and tested it using a grenade (on a test server).

Beakers:
1) 50u of that foam reagent stuff, and 50 CLF3
2) 50u of the foam reagent stuff, and 50 more CLF3

result;


husk was due to space damage.
the mob kept on lighting on fire - wasn't immediately killed or anything like that. it took time.
the mob was harmed by the burn damage extremely slowly, i was unable to tell whether they had reagents inside of them or not.
reagents moved extremely fast.

i'd say it's fairly powerful. I don't think we'll have any problems.

**Comment by Cruix** [comment]
Let's give it a try.

**Pull request merged**